projects/angular/src/forms/datepicker/date-container.ts

```diff
@@ -140,6 +170,29 @@ export class ClrDateContainer extends ClrAbstractContainer implements AfterViewI
     }
   }
 
+  @Input()
+  set rangeOptions(rangeOptions) {
+    this.dateIOService.setRangeOptions(rangeOptions);
+  }
+
+  @Input()
+  set min(dateString: string) {
+    this.dateIOService.setMinDate(dateString);
+    if (this.clrStartDateInput && this.clrEndDateInput) {
+      this.clrStartDateInput.triggerControlInputValidation();
+      this.clrEndDateInput.triggerControlInputValidation();
+    }
+  }
+
+  @Input()
+  set max(dateString: string) {
+    this.dateIOService.setMaxDate(dateString);
+    if (this.clrStartDateInput && this.clrEndDateInput) {
+      this.clrStartDateInput.triggerControlInputValidation();
+      this.clrEndDateInput.triggerControlInputValidation();
+    }
+  }
```

Comment:
```suggestion
  @Input()
  set min(dateString: string) {
    if (this.dateNavigationService.isRangePicker) {
      this.dateIOService.setMinDate(dateString);
      this.clrStartDateInput.triggerControlInputValidation();
      this.clrEndDateInput.triggerControlInputValidation();
    } else {
      console.error('Error! The date container [min] input only works for date range pickers. Use the native `min` attribute/property for single-date inputs.');
    }
  }

  @Input()
  set max(dateString: string) {
    if (this.dateNavigationService.isRangePicker) {
      this.dateIOService.setMaxDate(dateString);
      this.clrStartDateInput.triggerControlInputValidation();
      this.clrEndDateInput.triggerControlInputValidation();
    } else {
      console.error('Error! The date container [max] input only works for date range pickers. Use the native `max` attribute/property for single-date inputs.');
    }
  }
```

These inputs only work for the date range picker (validation is not triggered for single-date inputs in these setters), so I think the code should print a console error when used for a single-date input.